Last-Update: 2013-05-02
Forwarded: https://github.com/Feh/nocache/pull/11
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: man page corrections (lintianisation)
 hyphen-used-as-minus-sign

--- a/man/cachedel.1
+++ b/man/cachedel.1
@@ -1,9 +1,9 @@
 .TH CACHEDEL "1" "March 2013" "cachedel" ""
 .SH NAME
 cachedel \- drop pagecache for a file
 .SH SYNOPSIS
-cachedel [-n <n>] \fBfile\fR
+cachedel [\-n <n>] \fBfile\fR
 .SH OPTIONS
 .TP
 \fB\-n <n>\fR "Repeat system call"
 Will call posix_fadvise(POSIX_FADV_DONTNEED) \fB<n>\fR in a row.
--- a/man/nocache.1
+++ b/man/nocache.1
@@ -1,9 +1,9 @@
 .TH NOCACHE "1" "March 2013" "nocache" ""
 .SH NAME
 nocache \- don't use Linux page cache on given command
 .SH SYNOPSIS
-nocache [-n <n>] \fBcommand\fR [argument...]
+nocache [\-n <n>] \fBcommand\fR [argument...]
 .SH OPTIONS
 .TP
 \fB\-n <n>\fR "Set number of fadvise calls"
 Execute the `posix_fadvise` system call \fB<n>\fR times in a row.
